website/client/src/components/groups/groupFormModal.vue

```diff
@@ -616,14 +616,14 @@ export default {
           name: catName,
         });
       });
-      this.workingGroup.categories = serverCategories;
 
       const groupData = { ...this.workingGroup };
+      groupData.categories = serverCategories;
```

Comment:
If you rename `serverCategories` to `categories` you could do this in a single go like this:

```
const groupData = { ...this.workingGroup, categories };
```